src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -74919,6 +74919,29 @@ A M()
         return s; 
     }
 } 
+";
+            var comp = CreateCompilation(source);
+            comp.VerifyDiagnostics();
+        }
+
+        [WorkItem(28324, "https://github.com/dotnet/roslyn/issues/28324")]
```

Comment:
>https://github.com/dotnet/roslyn/issues/28324 [](start = 26, length = 45)

Looks like we have a test already: `NonNullTypes_GenericOverriddenMethod_ValueType`

Reply:
Okay. I will delete this and add the `WorkItem` notation to that one.

Reply:
Hhu. It's already tagged with that work item number. I'll just remove this test then.